src/main/logseq/api.cljs

```diff
@@ -678,9 +678,9 @@
 
 (def ^:export get_block
   (fn [id-or-uuid ^js opts]
-    (when-let [block (cond
-                       (number? id-or-uuid) (db-utils/pull id-or-uuid)
-                       (string? id-or-uuid) (db-model/query-block-by-uuid (sdk-utils/uuid-or-throw-error id-or-uuid)))]
+    (when-let [block (if (number? id-or-uuid)
```

Comment:
The bug is addressed here, all the other code changes are for testing.

Reply:
So the buggy point is:
https://github.com/logseq/logseq/pull/9354/files#diff-063cd1f27069a023acb43000c2bf850790f888edd7e61055583c3383a0a39d8fL706
sending a #uuid obj, while the `get_block` function doesn't support it?